docs/current_docs/integrations/_integration.mdx.template

```diff
@@ -0,0 +1,34 @@
+---
+slug: /integrations/$NAME_OF_INTEGRATION
```

Comment:
```suggestion
slug: /integrations/<name-of-integration>
```
Since the format of slugs seems to be lowercase kebab-case when more than one word, maybe this hints this better?
For example:
https://docs.dagger.io/integrations/azure-pipelines